scripts/poststart_hook.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# Configuration
+PLATFORM="docker"  # Change to "helm" or "cdk" as needed
+VERSION="unknown"
+STATUS=""
+SERVER_HEALTH_URL="http://hyperswitch-server:8080/health"
+HYPERSWITCH_URL="http://hyperswitch-server:8080/health/ready"
+WEBHOOK_URL="https://hyperswitch.gateway.scarf.sh/$PLATFORM"
+
+# Fetch health status
+echo "Fetching app server health status..."
+HEALTH_RESPONSE=$(curl -s --fail "$SERVER_HEALTH_URL") || HEALTH_RESPONSE="connection_error"
+
+if [ "$HEALTH_RESPONSE" = "connection_error" ]; then
+    STATUS="error"
+    ERROR_MESSAGE=$(echo "404 response" | jq -sRr @uri)
+    PARAMS="$PARAMS&status=$STATUS"
+    PARAMS="$PARAMS&error_message=$ERROR_MESSAGE"
```

Comment:
This can be slightly simplified:

```suggestion
    PARAMS="$PARAMS&status=$STATUS&error_message=$ERROR_MESSAGE"
```

Moreover, why aren't we using `--data-urlencode` like in the pre-start hook script? Can we please maintain consistency across both scripts?